.golangci.yml

```diff
@@ -56,207 +80,487 @@ output:
+    # go: "1.15"
+    # SAxxxx checks in https://staticcheck.io/docs/configuration/options/#checks
+    # Default: ["*"]
     checks: ["all"]
 
   errcheck:
-    # report about not checking of errors in type assertions: `a := b.(MyStruct)`;
-    # default is false: such cases aren't reported by default.
-    check-type-assertions: false
+    # Report about not checking of errors in type assertions: `a := b.(MyStruct)`.
+    # Such cases aren't reported by default.
+    # Default: false
+    check-type-assertions: true
 
-    # report about assignment of errors to blank identifier: `num, _ := strconv.Atoi(numStr)`;
-    # default is false: such cases aren't reported by default.
+    # report about assignment of errors to blank identifier: `num, _ := strconv.Atoi(numStr)`.
+    # Such cases aren't reported by default.
+    # Default: false
     check-blank: true
+
   govet:
-    # report about shadowed variables
-    check-shadowing: true
-
-    # Obtain type information from installed (to $GOPATH/pkg) package files:
-    # golangci-lint will execute `go install -i` and `go test -i` for analyzed packages
-    # before analyzing them.
-    # By default this option is disabled and govet gets type information by loader from source code.
-    # Loading from source code is slow, but it's done only once for all linters.
-    # Go-installing of packages first time is much slower than loading them from source code,
-    # therefore this option is disabled by default.
-    # But repeated installation is fast in go >= 1.10 because of build caching.
-    # Enable this option only if all conditions are met:
-    #  1. you use only "fast" linters (--fast e.g.): no program loading occurs
-    #  2. you use go >= 1.10
-    #  3. you do repeated runs (false for CI) or cache $GOPATH/pkg or `go env GOCACHE` dir in CI.
-    use-installed-packages: false
+    # Report about shadowed variables.
+    # Default: false
+    check-shadowing: false
+    # Disable all analyzers.
+    # Default: false
+    disable-all: false
+    # Enable analyzers by name (in addition to default).
+    # Run `go tool vet help` to see all analyzers.
+    # Default: []
+    # enable:
+    #   - asmdecl
+    #   - assign
+    #   - atomic
+    #   - atomicalign
+    #   - bools
+    #   - buildtag
+    #   - cgocall
+    #   - composites
+    #   - copylocks
+    #   - deepequalerrors
+    #   - errorsas
+    #   - fieldalignment
+    #   - findcall
+    #   - framepointer
+    #   - httpresponse
+    #   - ifaceassert
+    #   - loopclosure
+    #   - lostcancel
+    #   - nilfunc
+    #   - nilness
+    #   - printf
+    #   - reflectvaluecompare
+    #   - shadow
+    #   - shift
+    #   - sigchanyzer
+    #   - sortslice
+    #   - stdmethods
+    #   - stringintconv
+    #   - structtag
+    #   - testinggoroutine
+    #   - tests
+    #   - unmarshal
+    #   - unreachable
+    #   - unsafeptr
+    #   - unusedresult
+    #   - unusedwrite
```

Comment:
This list is noise only...  Available analyzers can be found as stated in the comment above:
>Run `go tool vet help` to see all analyzers.

Reply:
Removed default configuration options that can be found via calling a tool or from a website